app/views/job_postings/show.html.slim

```diff
@@ -1,6 +1,10 @@
 - content_for(:container_class, 'index-container')
 = render 'home/navbar'
 
+- content_for(:head_additions) do
+  link crossorigin="" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" rel="stylesheet" /
+  script crossorigin="" integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA==" src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
```

Comment:
Not sure if I like it inline, but it makes the application script a bit less heavy…